sass/directives/_sections.scss

```diff
@@ -0,0 +1,20 @@
+@mixin logos-section {
```

Comment:
This section seems very specific to Employer and the styles are very specific to this particular implementation of a logos section. A logo component would make sense but the styles are not all necessarily reusable

- The background color could change depending on what page and in what order the logos are placed
- Section padding has a `.container--employer-section` class in Employer that I haven't added everywhere because I haven't yet looked at whether the styles should be applied to all `.container` classes. I don't have an argument for why this doesn't belong in style base, except that when there's a decision on the `.container` class it will be easy to find everything that needs to be updated if all of those things are in Employer
- The headline styles will also be easier to update if they're all in Employer. I've asked for a story to overhaul existing typography and buttons with Susan's new designs, and at that time we can find all of the one-off type styles that we've changed in Employer. The centering and the space around headlines might also be inherited from typography, and so they don't need to be made into a mixin

I don't know if I'm making good arguments but I feel like all of this stylesheet belongs in Employer